test/integration/appinfra/appinfra_test.go

```diff
@@ -69,21 +73,28 @@ func TestAppInfra(t *testing.T) {
 				TeamName:        prefixServiceName,
 			}
 
+			remoteState := bootstrap.GetStringOutput("state_bucket")
+
+			backend_bucket := strings.Split(appFactory.GetJsonOutput("app-group").Get(fmt.Sprintf("%s\\.%s.app_cloudbuild_workspace_state_bucket_name", appName, suffixServiceName)).String(), "/")
+			backendConfig := map[string]interface{}{
+				"bucket": backend_bucket[len(backend_bucket)-1],
+			}
+
 			servicePath := fmt.Sprintf("%s/%s/envs/shared", appSourcePath, fullServiceName)
 			t.Run(servicePath, func(t *testing.T) {
 				t.Parallel()
 
 				vars := map[string]interface{}{
-					"project_id":                 servicesInfoMap[fullServiceName].ProjectID,
-					"region":                     region,
-					"env_cluster_membership_ids": env_cluster_membership_ids,
-					"buckets_force_destroy":      "true",
+					"remote_state_bucket":   remoteState,
```

Comment:
do we also need to add `remote_state_bucket` variable here?
 https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/5-appinfra/apps/default-example/hello-world/envs/shared/variables.tf